packages/validator/test/e2e/web3signer.test.ts

```diff
@@ -135,9 +135,7 @@ describe("web3signer signature test", function () {
     await assertSameSignature("signVoluntaryExit", pubkeyBytes, validatorIndex, epoch);
   });
 
-  // ValidatorRegistration includes a timestamp so it's possible that web3signer instance and local instance
-  // sign different messages and this test fails. Disabling unless it can be proven deterministic
-  it.skip("signValidatorRegistration", async () => {
+  it("signValidatorRegistration", async () => {
```

Comment:
The timestamp is part of the payload sent to the web3signer and will be respected, see [spec](https://github.com/ethereum/remote-signing-api/blob/7879ed3284bf88acb1149d30a70fbf8b554c183b/signing/paths/sign.yaml#L580-L588)

Changing the timestamp would also change the signing root which I would assume web3signer does not recalculate. Maybe this behavior was different in the past but the tests are consistenly passing now.